### PR TITLE
[7.26.x] Refactor KieServerS2iJbpmEJBTimerPerfIntegrationTest to use dedicated scenario

### PR DESCRIPTION
**Pull request by sutaakar**
@gmunozfe Can you please take a look? I need to refactor the test as generic scenario is going to be deleted.

### CODE REVIEW

File: test-cloud/test-cloud-performance/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iJbpmEJBTimerPerfIntegrationTest.java [hunk 1]
```diff
@@ -129,69 +133,59 @@
     
     protected QueryServicesClient queryServicesClient;
 
-    protected String repositoryName;
+    private static String gitRepositoryName = Git.getProvider().createGitRepositoryWithPrefix("KieServerS2iJbpmRepository", KieServerS2iJbpmEJBTimerPerfIntegrationTest.class.getResource(PROJECT_SOURCE_FOLDER).getFile());
```
Comment (gmunozfe) [inline test-cloud/test-cloud-performance/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iJbpmEJBTimerPerfIntegrationTest.java]: Can it be defined as protected?

File: test-cloud/test-cloud-performance/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iJbpmEJBTimerPerfIntegrationTest.java [hunk 2]
```diff
@@ -129,69 +133,59 @@
...
-        processServicesClient = KieServerClientProvider.getProcessClient(deploymentScenario.getKieServerDeployments().get(0));
-        queryServicesClient = KieServerClientProvider.getQueryClient(deploymentScenario.getKieServerDeployments().get(0));
+        processServicesClient = KieServerClientProvider.getProcessClient(deploymentScenario.getKieServerDeployment());
+        queryServicesClient = KieServerClientProvider.getQueryClient(deploymentScenario.getKieServerDeployment());
     }
 
     @After
-    public void deleteRepo() {
-        Git.getProvider().deleteGitRepository(repositoryName);
-        deploymentScenario.getKieServerDeployments().get(0).resetRouterTimeout();
```
Comment (gmunozfe) [inline test-cloud/test-cloud-performance/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iJbpmEJBTimerPerfIntegrationTest.java]: Is it not longer needed to reset the router timeout for the deployment?

File: test-cloud/test-cloud-performance/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iJbpmEJBTimerPerfIntegrationTest.java (hunk 1 above)
Comment (sutaakar) [reply test-cloud/test-cloud-performance/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iJbpmEJBTimerPerfIntegrationTest.java]: is there a specific reason to have it protected?

File: test-cloud/test-cloud-performance/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iJbpmEJBTimerPerfIntegrationTest.java (hunk 2 above)
Comment (sutaakar) [reply test-cloud/test-cloud-performance/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iJbpmEJBTimerPerfIntegrationTest.java]: Timer reset is done in resetRouterTimeout() below :)

Comment (gmunozfe) [reply test-cloud/test-cloud-performance/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iJbpmEJBTimerPerfIntegrationTest.java]: sorry, haven't seen it :)

File: test-cloud/test-cloud-performance/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iJbpmEJBTimerPerfIntegrationTest.java (hunk 1 above)
Comment (gmunozfe) [reply test-cloud/test-cloud-performance/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iJbpmEJBTimerPerfIntegrationTest.java]: probably not, leave it private is also fine, making all protected is just one of my quirks :-)

**Pull request merged**